src/Domains/Connectors/ScrapperApi/Actions/ScrapperAction.php

```diff
@@ -66,6 +67,10 @@ public function execute(): array
                     app: $this->app
                 );
                 $importerProducts++;
+
+                if (App::environment('local')) {
```

Comment:
@FredPeal only 1 result in local to avoid credit consumption ?

Reply:
yes